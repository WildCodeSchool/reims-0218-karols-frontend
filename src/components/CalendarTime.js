import React from "react"
import { Alert } from "reactstrap"

const CalendarTime = props => {
  return (
    <div>
      <Alert color="info">
        <a href="#" className="alert-link">
          09:00
        </a>
      </Alert>
    </div>
  )
}

export default CalendarTime
