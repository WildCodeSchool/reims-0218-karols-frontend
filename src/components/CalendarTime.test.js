import React from "react"
import ReactDOM from "react-dom"
import CalendarTime from "./CalendarTime"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<CalendarTime />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<CalendarTime />).toJSON()
  expect(tree).toMatchSnapshot()
})
