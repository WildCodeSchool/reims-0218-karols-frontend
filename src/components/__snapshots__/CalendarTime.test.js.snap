// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="justify-content-center"
>
  <div
    style={
      Object {
        "backgroundColor": "#ccecfa",
        "borderRadius": "4px",
        "cursor": "pointer",
        "fontSize": "14px",
        "fontWeight": "700",
        "height": "34px",
        "lineHeight": "34px",
        "margin": "10px 0",
        "position": "relative",
        "textAlign": "center",
        "width": "100%",
      }
    }
  >
     
    13:30
     
  </div>
</div>
`;
